functions/replay.fish

```diff
@@ -10,6 +10,7 @@ function replay --description "Run Bash commands replaying changes in Fish"
         case \*
             set --local env
             set --local sep @$fish_pid(random)(command date +%s)
+            set --local argv $argv[1] \"$argv[2..-1]\"
```

Comment:
What are we trying to do here?

Reply:
Fish was just passing all arguments together.
If you put for example: replay command a b "c" "d e" f
The script understand: command a b c d e f

This line make the script understand: command "a" "b" "c" "d e" "f"

Which should be the natural

Reply:
@jorgebucaran you must be worried about commands with --long or -shorts arguments, for example:
 ls "-a" "--color"; ping "-c" google.com; fdisk "-l"
but it also works that way.
Are there something else worrying you?

Reply:
What do you mean by backward?
If it's about "cd" command, it works:
cd ".."
cd "-"

Reply:
Meant to ask if this PR includes breaking changes.

Reply:
I'm sorry, what do you mean by breakin changes?
It works to:
echo a ; echo "b"
cat /etc/hosts | sort
cat "/etc/hosts" | sort
sort < /etc/hosts
sort < "/etc/hosts"
sort < "/etc/hosts" > txt
sort < "/etc/hosts" > "txt"
ls /folder 2> /dev/null

It's just separating the arguments

Reply:
Will merging this PR change how Replay behaves in an unexpected way? Say I have a script that uses Replay. Will I have to update said script after merging this?

Reply:
No, the PR is just fixing bugs

Now, if you: cd "System Information"
it gives the error: bash: line 2: cd: too many arguments
and/or: cd: The directory “System” does not exist

After the PR, it will be just as it should be. Each parameter considered as a parameter.